api/lib/utils.js

```diff
@@ -9,6 +9,7 @@ function renameCounterpartyToIssuer(amount) {
     amount.issuer = amount.counterparty;
     delete amount.counterparty;
   }
+  return amount;
```

Comment:
Reminds me of, https://aphyr.com/posts/320-returning-self-or-void-suggests-mutability